cmd/workspace/workspace/overrides.go

```diff
@@ -19,7 +27,36 @@ func exportOverride(exportCmd *cobra.Command, exportReq *workspace.ExportRequest
 	exportCmd.Annotations["template"] = `{{.Content | b64_decode}}`
 }
 
+func importOverride(importCmd *cobra.Command, importReq *workspace.Import) {
+	var filePath string
+	importCmd.Use = "import TARGET_PATH"
+	importCmd.Flags().StringVar(&filePath, "file", "", `Path of local file to import`)
+	importCmd.MarkFlagsMutuallyExclusive("content", "file")
+
+	originalRunE := importCmd.RunE
+	importCmd.RunE = func(cmd *cobra.Command, args []string) error {
+		if filePath != "" {
+			b, err := os.ReadFile(filePath)
+			if err != nil {
+				return err
+			}
+			importReq.Content = base64.StdEncoding.EncodeToString(b)
+		}
+		err := originalRunE(cmd, args)
```

Comment:
Because this wraps instead of overrides the original (generated) impl, the test coverage should include cases where we don't specify `--file` as well.

Reply:
done